src/components/slider/slider.scss

```diff
@@ -137,6 +148,58 @@ md-slider *::after {
   border-color: md-color($md-accent);
 }
 
+.md-slider-thumb-label {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  position: absolute;
+  left: -($md-slider-thumb-label-height / 2);
+  top: $md-slider-thumb-label-top;
+  width: $md-slider-thumb-label-width;
+  height: $md-slider-thumb-label-height;
+  border-radius: max($md-slider-thumb-label-height, $md-slider-thumb-label-width);
```

Comment:
To clarify, your method would result in a sort of pill-shape when these values do not match.  If this is desired, I think what you actually want is half of the smaller dimension: `border-radius: min($md-slider-thumb-label-height, $md-slider-thumb-label-width) / 2`.  Your method will work too, but that's because the browser caps the border radius at this value.

If you wanted more of an oval shape, you can also simplify it to `border-radius: 50%`